local/src/main/java/com/project/local/impl/LocalDataSourceImpl.kt

```diff
@@ -1,35 +1,32 @@
 package com.project.local.impl
 
 import com.project.data.local.MaydayUserLocalDataSource
-//import com.project.local.model.LocalDao
+import com.project.data.model.LocalItem
+import com.project.local.model.LocalDao
+import com.project.local.model.MayDayRoomEntity
+import io.reactivex.Completable
+import io.reactivex.Single
+import io.reactivex.schedulers.Schedulers
 
 
-/***
- * 코드는 다 지우기 애매해서 주석처리 해놓았습니다.
- *
- *
- * 20200724 -> 메서드명 바꿨는데 애매하시면 지우셔도 됩니다.
- ***/
+class LocalDataSourceImpl(private val localItemDao: LocalDao) : MaydayUserLocalDataSource {
 
-//class LocalDataSourceImpl(private val localItemDao: LocalDao) : MaydayUserLocalDataSource {
+    override fun saveList(data: LocalItem): Completable {
+        return localItemDao.insert(
+            MayDayRoomEntity(
+                id = System.currentTimeMillis(),
+                list = data.items
+            )
+        )
+    }
 
-//    override fun saveList(data: Entity): Completable {
-//        return localItemDao.insert(
-//            LocalItem(
-//                id = System.currentTimeMillis(),
-//                list = data.items
-//            )
-//        )
-//    }
-//
-//
-//    override fun getList(): Single<Entity> =
-//        localItemDao
-//            .getList().map { Entity(items = it.list) }
-//            .toSingle()
-//            .subscribeOn(Schedulers.io())
-//
-//    override fun deleteList() {
-//        localItemDao.deleteAll()
-//    }
-//}
\ No newline at end of file
+    override fun getList(): Single<LocalItem> =
```

Comment:
로컬 db에는 데이터가 없을수도 있는 경우가 있으니 Single 보단 Maybe 가 좋을듯해요

```suggestion
override fun getList() : Maybe<LocalItem> =
```

Reply:
그렇네요 감사합니다 수정하였습니다
d38b5f1